src/pages/about.js

```diff
@@ -152,16 +163,21 @@ class App extends React.Component {
 
           <p>
             Special thanks to{" "}
-            <a href="https://netlify.com" target="_blank">
-              Netlify
+            <a href="https://blackswho.design" target="_blank">
+              Blacks Who Design
+            </a>{" "}
+            for inspiration, and to{" "}
+            <a href="https://womenwho.design" target="_blank">
+              Women Who Design
             </a>{" "}
-            for their support.
+            for allowing similar sites to fluorish
+            by open-sourcing their code!
           </p>
           <p>
-            ✌️{" "}
```

Comment:
doesn't show up very well in my web browser, but this is a ✌️ — I did consider leaving it in but adding the skin-tone (e.g. ✌🏿), so let me know if that feels better

Reply:
I feel like this alright to remove. I don't think it adds significant value to the page